light_controller/ir_toy/ir_toy.py

```diff
@@ -0,0 +1,351 @@
+# -*- coding: utf-8 -*-
+import logging
+import time
+import struct
+
+import serial
+
+from .ir_toy_settings import IR_ToySettings
+
+__author__ = 'akm'
+
+
+class CommandEnum:
+    reset = 0x00
```

Comment:
the enum naming convention should be consistent with Camel case or camel case or all UPPER_CASE